osu.Game.Rulesets.Osu/Difficulty/OsuPerformanceCalculator.cs

```diff
@@ -51,18 +51,32 @@ public override PerformanceAttributes Calculate()
             if (mods.Any(m => m is OsuModSpunOut) && totalHits > 0)
                 multiplier *= 1.0 - Math.Pow((double)Attributes.SpinnerCount / totalHits, 0.85);
 
-            if (mods.Any(h => h is OsuModRelax))
-            {
-                // As we're adding Oks and Mehs to an approximated number of combo breaks the result can be higher than total hits in specific scenarios (which breaks some calculations) so we need to clamp it.
-                effectiveMissCount = Math.Min(effectiveMissCount + countOk + countMeh, totalHits);
```

Comment:
Why did you remove this? In my opinion 100/50s being part of miss penalty makes sense

Reply:
Accuracy calculation is added and scaled therefore I don't see this as required. Me and apollo have already discussed possible similar adjustments into accuracy calculation itself.

Reply:
I would personally prefer having 100/50s being part of the miss penalty, for example as a 0.66/0.33 additions instead of reintroducing accuracy which in itself is a tapping concept.

Reply:
Since I'm going to remove accuracy PP, I'll go ahead and do this too.

Reply:
Would be cool if you discussed things somewhere public btw, either in pp dev or osu dev

Reply:
I'm in both servers, I had access to write in osu dev but it got removed for some reason. I'll talk about it in pp dev though since apollo invited me there